Map: document namespace conflict between Controls.Maps.Map and ApplicationModel.Map

## Summary

The Map documentation does not mention a common compile error caused by an ambiguous reference between `Microsoft.Maui.Controls.Maps.Map` and `Microsoft.Maui.ApplicationModel.Map`. Both are commonly imported via implicit usings, and the conflict is not obvious from the error message.

## Why it matters

When a developer adds the Maps NuGet package and writes `new Map()` or uses `<Map>` in XAML, they get:

```
error CS0104: 'Map' is an ambiguous reference between 'Microsoft.Maui.Controls.Maps.Map' and 'Microsoft.Maui.ApplicationModel.Map'
```

This is confusing because `ApplicationModel.Map` (used for `Map.OpenAsync` to launch the native maps app) is auto-imported and invisible. Developers unfamiliar with C# namespace aliases may not know how to resolve it.

## What should be documented

Add a note early in the Map setup documentation:

> **Note:** `Microsoft.Maui.Controls.Maps.Map` can conflict with `Microsoft.Maui.ApplicationModel.Map` (which provides `Map.OpenAsync` for launching the native maps app). If you get an ambiguous reference error, add a namespace alias:

```csharp
// C# — add at the top of files that use the Map control
using Map = Microsoft.Maui.Controls.Maps.Map;
```

```xml
<!-- XAML — use a namespace prefix -->
<ContentPage xmlns:maps="clr-namespace:Microsoft.Maui.Controls.Maps;assembly=Microsoft.Maui.Controls.Maps">
    <maps:Map x:Name="map" />
</ContentPage>
```

Also note that **Windows does not have native Map support** in `Microsoft.Maui.Controls.Maps` — Windows apps must use `CommunityToolkit.Maui.Maps` instead.

## Suggested location

`docs/user-interface/controls/map.md` — in the setup/getting-started section, before the first code example

--- a/docs/user-interface/controls/map.md
+++ b/docs/user-interface/controls/map.md
@@ -1,7 +1,7 @@
 ---
 title: "Map"
 description: "Learn how to use the Map control, which is a cross-platform view for displaying and annotating maps. The Map control is available in the Microsoft.Maui.Controls.Maps NuGet package."
-ms.date: 08/30/2024
+ms.date: 03/23/2026
 ---
 
 # Map
@@ -44,6 +44,15 @@ public static class MauiProgram
 ```
 
 Once the NuGet package has been added and initialized, <xref:Microsoft.Maui.Controls.Maps.Map> APIs can be used in your project.
+
+> [!NOTE]
+> <xref:Microsoft.Maui.Controls.Maps.Map> can conflict with <xref:Microsoft.Maui.ApplicationModel.Map> (which provides `Map.OpenAsync` for launching the native maps app). Both types are commonly available via implicit usings, and using `Map` in code can produce an `error CS0104: 'Map' is an ambiguous reference` compiler error. To resolve this, add a namespace alias in any C# file where the conflict occurs:
+>
+> ```csharp
+> using Map = Microsoft.Maui.Controls.Maps.Map;
+> ```
+>
+> You can then use `Map` to refer to the controls map throughout that file.
 
 ### Platform configuration
 

